_data/organizers/organizing_committee.yml

```diff
@@ -0,0 +1,164 @@
+  linkedin_url: https://www.linkedin.com/in/nicolelcarlson/
+  github_url: https://github.com/parsing-science/
+  image: /assets/images/organizers/nicole_carlson.jpg
+  website_url: http://www.parsingscience.com/
+  
+- name: Aaron Richter
+  desc: Aaron is a Senior Data Scientist at Saturn Cloud, and co-organizer of PyData Miami. He works to make other data scientists faster and happier!
+  twitter_url: https://twitter.com/rikturr
+  linkedin_url: https://linkedin.com/in/rikturr
+  github_url: https://github.com/rikturr
+  website_url: https://rikturr.com/
+  image: /assets/images/organizers/aaron_richter.jpg
+
+- name: Conrad Ho
+  desc: Conrad is a partner at PythonAnywhere, a startup hoping to democratize access to programming via an online platform to write and run Python code. He is a PyData London organiser, surfer, photographer, chess player.
+  twitter_url: 
+  linkedin_url: 
+  github_url: 
+  website_url: 
+  image: /assets/images/organizers/conrad_ho.jpg
+
+- name: Sebastian Neubauer
+  desc: Sebastian joined Blue Yonder as a Data Scientist after completing his PhD in physics. One of the first things Sebastian learned on the job is that it takes a lot more than just a Jupyter notebook to deliver value to customers. Sebastian was one of the main organizers of the PyCon.DE & PyData 2017 & 2018 in Karlsruhe and has been a member of the PyCon.DE organizers committee since then.
+  twitter_url: https://twitter.com/sebineubauer
+  linkedin_url: https://www.linkedin.com/in/sebastian-neubauer-16626a79
+  github_url: https://github.com/sebastianneubauer
+  website_url: https://tech.blueyonder.com/
+  image: /assets/images/organizers/sebastian_neubauer.jpeg
+
+- name: Marco Bonzanini
+  desc: Marco is a Data Science consultant and trainer based in London (UK), organiser of PyData London
+  twitter_url: https://twitter.com/marcobonzanini
+  linkedin_url: https://www.linkedin.com/in/marcobonzanini/
+  github_url: https://github.com/bonzanini
+  website_url: https://marcobonzanini.com/
+  image: /assets/images/organizers/marco_bonzanini.jpeg
+
+- name: Motasim Rahman Kazmi
+  desc: Motasim is a CS undergrad and a full stack developer who loves data. He is an Intel Software Innovator and currently on the lookout for new and exciting opportunities. He also helps in organizing PyData Delhi meetups and conferences.
+  linkedin_url: https://www.linkedin.com/in/mrk333/
+  github_url: https://github.com/mrkazmi333
+  image: /assets/images/organizers/motasim_kazmi.jpg
+
+- name: Chetan Chawla
+  desc: Chetan is a data analyst and an astrophysics enthusiast. He builds stuff to solve real world problems, loves space and plays piano.
+  twitter_url: https://twitter.com/chetan_chawla_
+  linkedin_url: https://linkedin.com/in/chetan-chawla
+  github_url: https://github.com/chetanchawla
+  website_url: https://sites.google.com/view/chetanchawla
+  image: /assets/images/organizers/chetan_chawla.jpg
+
+- name: Sanket Verma
```

Comment:
Let's move Sanket and Matthijs to conference_chair.yml, with title 'Executive Vice-chair'

Reply:
Sure.